arcsi/api/utils.py

```diff
@@ -158,6 +158,30 @@ def broadcast_audio(
     return False
 
 
+def cleanup_show_playlist(broadcast_playlist):
+    az = AzuraArchive(
```

Comment:
Did you think about putting these two to the AA object as its methods? Then you would not need to instantiate these again here.